### DISCUSSION TITLE
What outfit is this

### DISCUSSION THREAD

**Discussion by PongMaster69** [Monika-After-Story/MonikaModDev#7772] [QA - tech support + general] [answered]
I was told to make a new issue. It's #7707.

**Comment by fromulus** [comment]
It's not one of the official spritepacks distributed here, so you'd have a better shot asking on [the mod's subreddit](https://www.reddit.com/r/MASFandom/) or on [the dedicated discord server](https://discord.gg/3kUGG57A).

**Comment by RaitoninguHikari** [comment]
@PongMaster69  
Sorry for the late reply, the jacket is an extra sprite that was posted a long time ago by orcaramelo, I just checked the link is down so I believe it deleted or removed it for some reason, it would be easier to find by reddit as quoted above.

**Comment by lunulae** [comment]
If this is the jacket Orca made, it was created as a private commission and wasn't intended for public use. But to my recollection that jacket didn't include gloves, so this might be a different outfit.
I haven't got much experience searching the reddit so you might find it faster than I do, but if I see it I will update this comment.

**Reply by Teddy-James** [reply]
Thanks for the clarification! That makes sense if it was a private commission. I’ll keep digging around — I’m especially interested if gloves were involved, so that could explain the mix-up. Appreciate you keeping an eye out! And if you’re into quality gloves, [here’s](https://philleywood.com/products/harry-leather-gloves-black) something worth checking out!

**Comment by Rai99** [comment]
That, to my knowledge is a public spritepack made by dnsgfs and can be found [here](https://drive.google.com/drive/u/0/folders/1eEca2kOo4LTiZisYYBdeqyjoa6K9Lt40) under `dnsgfs-jacket-leather.zip`, but I don't think it includes the gloves.

**Comment by SorcererPug** [comment, marked as answer]
This is indeed the jacket Orca made, had to do a side-by-side to verify that unless it's a really good copy.

The user who originally commissioned it (u/bbrandis) on Reddit [released it publicly](https://www.reddit.com/r/MASFandom/comments/mkjv6g/orcas_leather_jacket_sprite_packs_repost/?utm_source=share&utm_medium=web2x&context=3) about 5 months ago.
